src/components/Footer.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "styled-components";
+
+const FooterWrapper = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: -webkit-fill-available;
```

Comment:
This property only works for WebKit
<img width="1280" alt="Screen Shot 2019-07-30 at 10 42 22" src="https://user-images.githubusercontent.com/14864439/62114834-b790f300-b2b7-11e9-8911-fe77ebf0eee8.png">
 browsers. On Firefox, for instance, the footer is messed up:

One solution here is to use the `width: calc(100%  - 256px)`. Which is basically the total width minus the side paddings.

Reply:
Hey Fernando, thanks for the comments!

I was having a hard time struggling a little bit with the height/width structure with padding not being counted inside this measurements I guess.

I fixed these problems by setting the #root with a flex display, so that the _main_ can grow to whatever side it needs to grow. This way, the Footer is always displayed at the bottom, and not fixed to the page anymore.

I also had to make the height of the Timestamp's  _Card_ a little bit flexible, so that it does not always take the whole size of the _main_, this way the _Card_'s height is set to 400px and goes back to "auto" if there is any item added to the form.

Reply:
Home view:
![image](https://user-images.githubusercontent.com/9327569/62129817-046fcc00-b2ae-11e9-9e73-216336781f8c.png)

Added file to the form:
![image](https://user-images.githubusercontent.com/9327569/62129853-1487ab80-b2ae-11e9-8a07-22a7872b8b8a.png)

Results page also looks good:
![image](https://user-images.githubusercontent.com/9327569/62129899-34b76a80-b2ae-11e9-8357-dd00e87f593f.png)